app/GUI.py

```diff
@@ -1,49 +1,356 @@
-from datetime import date
+import logging
+from collections import defaultdict
+
+from PyQt5 import QtCore, QtWidgets
+from PyQt5.QtCore import Qt
+from PyQt5.QtGui import QColor
+from PyQt5.QtGui import QFont
+from PyQt5.QtWidgets import QTableWidgetItem, QLabel, QMessageBox, QPushButton
 
 from app import APP
-from core.domain.porocilo import RacunskoPorocilo
 
+log = logging.getLogger(__name__)
+
+
+class Ui_MainWindow(object):
+	def setupUi(self, MainWindow):
+		MainWindow.setObjectName("MainWindow")
+		MainWindow.resize(1037, 900)
+
+		self.centralwidget = QtWidgets.QWidget(MainWindow)
+		self.centralwidget.setObjectName("centralwidget")
+
+		# Nastavi začetne datume
+		default_start_date = QtCore.QDate(2023, 5, 2)
+		default_end_date = QtCore.QDate(2023, 5, 9)
+
+		self.exit_button = QPushButton("Izhod", self.centralwidget)
+		self.exit_button.setGeometry(900, 800, 75, 30)
+		self.exit_button.clicked.connect(MainWindow.close)
+
+		self.start_date_edit = QtWidgets.QDateEdit(self.centralwidget)
+		self.start_date_edit.setGeometry(QtCore.QRect(150, 50, 150, 30))
```

Comment:
Ti si celoten GUI sestavil programersko vendar to je malce mazohistično... Ponavadi to narediš na tak način...
https://youtu.be/FVpho_UiDAY?t=228